examples/server/server.cpp

```diff
@@ -3509,6 +3516,21 @@ int main(int argc, char ** argv) {
         }
         data["input_extra"] = input_extra; // default to empty array if it's not exist
 
+        std::string prompt = json_value(data, "prompt", std::string());
+        std::vector<llama_tokens> tokenized_prompts = tokenize_input_prompts(ctx_server.ctx, prompt, true, true);
```

Comment:
We should probably return an error if there is more than 1 resulting prompt?

Reply:
Because above we already checked if `data["prompt"]` is string, here we can be sure that we only have one single prompt to deal with. Probably an GGML_ASSERT here make more sense?

(The expected behavior of `tokenize_input_prompts` is that if prompt is a string, then output vector size is == 1)

Reply:
Got it. It's fine as it is.